plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplates.java

```diff
@@ -32,9 +32,10 @@
    * @param monitor progress monitor
    * @param name directory from which to load template
    */
-  // todo: config details are going to vary based on type of template; need a more generic
+  // TODO config details are going to vary based on type of template; need a more generic
   // solution such as key-value map or a different design
-  // todo what if project isn't empty?
+  // TODO what if project isn't empty?
+  // TODO is the folder hierarchy optimum for non-maven projects?
```

Comment:
I think this is good enough for now. We can fix it later if we need to. Please remove todo.